stable/distributed-tensorflow/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for distributed-tensorflow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+worker:
+  number: 2
+  podManagementPolicy: Parallel
+  image:
+    repository: cheyang/distributed-tf
```

Comment:
Let's discuss images again. Can't you use official ones?

Reply:
Because they only provides official Dockerfile but not offical docker image. Please check https://github.com/tensorflow/ecosystem/tree/master/docker. I have to build it by myself. Thanks.

Reply:
What about these?
https://hub.docker.com/r/tensorflow/tensorflow/

Reply:
The key is to put distributed python code https://github.com/tensorflow/ecosystem/blob/master/docker/mnist.py into the docker image. But official TensorFlow doesn't have this.